pkg/skaffold/runner/dev.go

```diff
@@ -63,6 +63,7 @@ func (r *SkaffoldRunner) doDev(ctx context.Context, out io.Writer) error {
 		}
 		fallthrough
 	case r.changeSet.needsRedeploy:
+		r.forwarderManager.Stop()
```

Comment:
would this gaurantee portforward is stopped when dev context is cancelled when user presses "cntrl c"?

Reply:
yah, so each go routine created by portforwarding depends on the `ctx context.Context` passed in to this function, so when the user presses "cntrl-c" they should all be stopped.